docs/multisig-cli.md

```diff
@@ -0,0 +1,390 @@
+# Multisig CLI Documentation
+
+The Holograph Multisig CLI is a TypeScript tool that generates Gnosis Safe Transaction Builder compatible JSON for common protocol operations. It provides automatic optimization, Tenderly simulation, and comprehensive error handling.
+
+## Overview
+
+The CLI handles three main operation types:
+1. **ETH → HLG → StakingRewards** batch transactions (primary use case)
+2. **Direct HLG deposits** to StakingRewards (when Safe already holds HLG)
+3. **StakingRewards ownership management** (transfer and acceptance)
+
+## Installation and Setup
+
+### Prerequisites
+
+```bash
+# Required: Node.js 18+ and npm
+npm install
+
+# Required: Environment configuration
+cp .env.example .env
+# Edit .env with your configuration
+```
+
+### Environment Variables
+
+| Variable | Required | Description | Default |
+|----------|----------|-------------|---------|
+| `MULTISIG_ADDRESS` | Yes | Gnosis Safe contract address | |
+| `SAFE_OWNER_ADDRESS` | Yes | Primary Safe owner for simulation | |
+| `SAFE_OWNER_ADDRESS_2` | No | Secondary Safe owner | |
+| `SLIPPAGE_BPS` | No | Slippage tolerance in basis points | 5000 (50%) |
+| `TENDERLY_ACCOUNT` | No | Tenderly account name | |
+| `TENDERLY_PROJECT` | No | Tenderly project name | |
+| `TENDERLY_ACCESS_KEY` | No | Tenderly API access key | |
+| `REQUIRED_FEE_TIER` | No | Force specific Uniswap fee tier | |
+| `PREFER_FEE_TIER` | No | Prefer fee tier with fallback | |
+
+### Example .env Configuration
+
+```bash
+# Safe Configuration
+MULTISIG_ADDRESS=0x8FE61F653450051cEcbae12475BA2b8fbA628c7A
+SAFE_OWNER_ADDRESS=0x1ef43b825f6d1c3bfa93b3951e711f5d64550bda
+SAFE_OWNER_ADDRESS_2=0x2ef43b825f6d1c3bfa93b3951e711f5d64550bdb
+
+# Trading Configuration  
+SLIPPAGE_BPS=2000  # 20% slippage tolerance
+
+# Tenderly Integration (optional but recommended)
+TENDERLY_ACCOUNT=attar
+TENDERLY_PROJECT=holograph
+TENDERLY_ACCESS_KEY=your_access_key_here
+
+# Uniswap Configuration (optional)
+PREFER_FEE_TIER=3000  # Prefer 0.3% pools
+```
+
+## Usage
+
+### Basic Commands
+
+```bash
+# Convert ETH to HLG and stake (default: 0.6 ETH)
+npm run multisig-cli
+
+# Convert specific amount
+npm run multisig-cli 0.5
+npm run multisig-cli --amount 0.1
+
+# Direct HLG deposit (no swapping)  
+npm run multisig-cli --hlg 1000
+
+# Simulation only (no JSON output)
+npm run multisig-cli --simulate-only 0.1
+
+# Ownership management
+npm run multisig-cli --transfer-ownership
+npm run multisig-cli --accept-ownership
```

Comment:
Should we have a command to pause and unpause?

Reply:
yes that would be nice

Reply:
done